src/search/search_algorithm.h

```diff
@@ -36,8 +36,7 @@ enum class UnsolvabilityVerificationType {
     PROOF_DISCARD
```

Comment:
The guard names at the top of this file should be renamed to SEARCH_ALGORITHM_H
(Sorry, I can't comment at the top of the file, probably since nothing changed there.)